wcfsetup/install/files/lib/page/SearchResultPage.class.php

```diff
@@ -247,4 +217,12 @@ protected function initObjectList()
     protected function readObjects()
     {
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function __run()
+    {
+        throw new IllegalLinkException();
```

Comment:
As this controller is effectively non-functional you could remove it outright. Compatibility with code relying on this is broken anyway.

Reply:
It's still used here: https://github.com/WoltLab/WCF/blob/3cfed0e1c3883117ff3d44e65655429a1d3b4b54/wcfsetup/install/files/lib/system/search/SearchResultHandler.class.php#L164